src/Dropdown/Dropdown.js

```diff
@@ -485,6 +486,7 @@ Dropdown.propTypes = {
   // Transition lengths
   transitionEnterTimeout: PropTypes.number,
   transitionLeaveTimeout: PropTypes.number,
+  trigger: PropTypes.object,
```

Comment:
Shall we use `PropTypes.element` instead? It won't work with _any_ object, right?